Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -935,6 +968,16 @@ theorem mem_closure_one_iff_norm {x : E} : x ∈ closure ({1} : Set E) ↔ ‖x
 theorem closure_one_eq : closure ({1} : Set E) = { x | ‖x‖ = 0 } :=
   Set.ext fun _x => mem_closure_one_iff_norm
 
+section Instances
+
+instance [SeminormedGroup E] : ContinuousENorm E where
+  continuous_enorm := continuous_enorm'--ENNReal.continuous_coe.comp continuous_nnnorm'
```

Comment:
Can you `@[to_additive]` this? Not having that will cause failures.

Reply:
This almost certainly causes the error you posted

Reply:
I had tried that before. If I do
```
@[to_additive]
instance [SeminormedGroup E] : ContinuousENorm E where
  continuous_enorm := ENNReal.continuous_coe.comp continuous_nnnorm'
```
I get an error `to_additive: can't transport instContinuousENorm to itself`.

The main doc-string of to_additive doesn't mention that error. What's the correct fix?

Reply:
I (somewhat blindly) had tried putting `to_additive` on `ContinuousENorm`; that also errors.

Reply:
> I get an error to_additive: can't transport instContinuousENorm to itself.

The error should be improved, I guess! It means that the additivised name is the same as the multiplicative name. The correct fix here is to correctly name the instance, namely `SeminormedGroup.toContinuousENorm`

Reply:
Thanks for explaining. Naming the instance in a better way works.